core/crypto/crypto.c

```diff
@@ -858,6 +859,31 @@ TEE_Result crypto_acipher_x25519_shared_secret(struct x25519_keypair
 }
 #endif
 
+#if !defined(CFG_DRV_X448)
```

Comment:
Shouldn't this be `CFG_CRYPTO_X448`?

Reply:
Yes，we fix it，thank you！

Reply:
Use CFG_CRYPTO_X448 might need add X448 implement in libTomCrypt ？